### PR TITLE
Fix namespace on git remote from https.

### PR DESCRIPTION
**Pull request by matheusgr**

### CODE REVIEW

**Review by lucis — APPROVED** [review]

File: utils/namespace.ts [hunk 1]
```diff
@@ -7,9 +7,18 @@ const namespaceFromGit = async (): Promise<string | undefined> => {
...
+    // HTTPS
+    // Fetch URL: https://github.com/deco-sites/example
+    regex = /.*https:\/\/github\.com\/(.+)/;
+  }
+
+  const result = fetchUrlLine.match(regex);
+  if (!result || (result.length < 1)) {
+    return undefined;
+  }
+  return "/" + result[1].trimEnd();
```
Comment (mcandeia) [inline utils/namespace.ts:21]: Not sure but I think the namespace should not start with `/` it should be something like `deco-sites/fashion`

**Review by mcandeia — CHANGES_REQUESTED** [review]

File: utils/namespace.ts (hunk 1 above)
Comment (matheusgr) [reply utils/namespace.ts:21]: I kept the current implementation, but it breaks import_map as is.

**Review by mcandeia — APPROVED** [review]

**Comment by mcandeia** [comment]
@matheusgr reverti pq quando fuit estar no mac meu import_map botou um undefined la
image


A gente ve junto hoje!

**Comment by mcandeia** [comment]
image
essa é a saida do meu comando

**Pull request merged**